src/client/Microsoft.Identity.Client/ConfidentialClientApplication.cs

```diff
@@ -69,7 +69,8 @@ public sealed partial class ConfidentialClientApplication
         /// </remarks>
         public AcquireTokenByAuthorizationCodeParameterBuilder AcquireTokenByAuthorizationCode(
             IEnumerable<string> scopes,
-            string authorizationCode)
+            string authorizationCode,
+            string pkceCodeVerifier = null)
```

Comment:
No, we will only be adding it via  the `With` method.

Reply:
Yes. this is dead code. removing